README.md

```diff
@@ -0,0 +1,54 @@
+## CUBRID Database Management System
+CUBRID is a DBMS being supported by an active community of open source developers 
+and provides better performance and features necessary for Web services. 
+
+This Software is released under Apache License 2.0 and BSD according to CUBRID components.
+For brevity, CUBRID Server Engine is under Apache License 2.0 and CUBRID APIs and Connectors are under BSD License.
+For details, please refer to the CUBRID License Page(http://www.cubrid.org/cubrid).
+
+Below You will see the brief list of sections to guide You to easily get started. 
+
+## MAJOR REFERENCES
+- CUBRID Official Site: http://www.cubrid.org ,  http://www.cubrid.com
+- CUBRID Development Site(Global): http://jira.cubrid.org
+- CUBRID Manuals: http://www.cubrid.org/manuals 
+- CUBRID CCI Manuals (V11.0) : https://www.cubrid.org/manual/en/11.0/api/cci.html (Eng)
+  https://www.cubrid.org/manual/ko/11.0/api/cci.html (Kor)
+
+## DOWNLOADS and FILE REPOSITORIES
+- http://www.cubrid.org/downloads
+- http://ftp.cubrid.org
+## HOW TO BUILD CUBRID CCI Driver
+### CUBRID CCI Driver Source Build Guide
+  * Requirement
+    - GNU Developer Toolset 8 or higher (Linux)
+    - MSVC 2017 (Windows)
+    - CMake (Linux, Windows)

```

Comment:
I think Git is required also
- Git (Linux, Windows) 1.7.6 or higher

Reply:
added
https://github.com/CUBRID/cubrid-cci/pull/29/commits/4b3a24d4bb122512e1afe26869c1a078acab6f40